flink-python/pyflink/pyflink_gateway_server.py

```diff
@@ -256,6 +256,8 @@ def construct_test_classpath():
         "flink-formats/flink-json/target/flink-json*.jar",
         "flink-python/target/artifacts/testDataStream.jar",
         "flink-python/target/flink-python*-tests.jar",
+        ("flink-state-backends/flink-statebackend-rocksdb/target/"
+         "flink-statebackend-rocksdb*tests.jar"),
```

Comment:
I added a test jar for pyflink test, for loading the `RocksDBStateBackendConfigTest$TestOptionsFactory` in test_get_set_options from test_state_backend.py. @HuangXingBo would you please to take a brief look?

Reply:
The change looks good to me.